addons/website/static/tests/interactions/snippets/form.test.js

```diff
@@ -187,12 +190,12 @@ test("(subject) form checks conditions", async () => {
     checkField(subjectEl, false, false);
     // Submit
     await click("a.s_website_form_send");
-    checkField(subjectEl, false, false);
+    checkField(subjectEl, false, true);
```

Comment:
Now that model-required fields are always enabled, there is always an error if they are not filled, even if they are conditionally hidden.